core/lib/spree/core/engine.rb

```diff
@@ -1,6 +1,7 @@
 # frozen_string_literal: true
 
 require 'spree/config'
+require 'spree/event/mailer_processor'
```

Comment:
if we plan to have multiple event processors maybe we can change the folders/modules structure to accomodate this? Something like: `spree/event/processors/mailer`. WDYT?

Reply:
👍makes sense